projects/media-viewer/src/lib/viewers/pdf-viewer/side-bar/bookmarks/bookmarks.component.ts

```diff
@@ -118,6 +117,7 @@ export class BookmarksComponent implements OnInit, OnDestroy {
 
     this.goToDestination.emit([
       bookmark.pageNumber,
+      // @TODO: Should the name be hard coded?
```

Comment:
yes, it's more of a type than a name really

Reply:
oh, I see